config.json

```diff
@@ -746,7 +1011,18 @@
         "slug": "raindrops",
         "name": "Raindrops",
         "uuid": "4522b139-6f0c-4172-a4fa-6a6adef33a9e",
-        "prerequisites": [],
+        "prerequisites": [
+          "integers",
+          "multiple-clause-functions",
+          "guards",
+          "if",
+          "strings",
+          "maps",
+          "enum"
+        ],
+        "practices": [
+          "enum"
```

Comment:
There are many different ways of solving this exercise, and they don't have much in common, not even `enum`. But I don't know what else to put there.